srtcore/srt.h

```diff
@@ -733,10 +793,14 @@ SRT_API int srt_epoll_update_ssock(int eid, SYSSOCKET s, const int* events);
 
 SRT_API int srt_epoll_wait(int eid, SRTSOCKET* readfds, int* rnum, SRTSOCKET* writefds, int* wnum, int64_t msTimeOut,
                            SYSSOCKET* lrfds, int* lrnum, SYSSOCKET* lwfds, int* lwnum);
-typedef struct SRT_EPOLL_EVENT_
+typedef struct SRT_EPOLL_EVENT_STR
 {
     SRTSOCKET fd;
     int       events; // SRT_EPOLL_IN | SRT_EPOLL_OUT | SRT_EPOLL_ERR
+#ifdef __cplusplus
+    SRT_EPOLL_EVENT_STR(SRTSOCKET s, int ev): fd(s), events(ev) {}
+    SRT_EPOLL_EVENT_STR() {} // NOTE: allows singular values, no init.
+#endif
 } SRT_EPOLL_EVENT;
```

Comment:
Changing typedef name in SRT API. Losing backward compatibility?

Reply:
The type name for `SRT_EPOLL_EVENT_` has always been a stub. People should use `SRT_EPOLL_EVENT`. OTOH if you think it's too risky, it can be reverted - the C++ version is still allowed to use the "typedefed" version of the name even in constructor.